EFAuditer/AuditDatabaseContext.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Newtonsoft.Json;
+
+namespace EFAuditer
+{
+    public class AuditDatabaseContext : DbContext
+    {
+        public AuditDatabaseContext(DbContextOptions<AuditDatabaseContext> options) : base(options) {}
+
+        public virtual DbSet<AuditLog> AuditLogs { get; set; }
+
+        public async Task AuditReadOperationAsync(string primaryKeyName, int primaryKeyId, object model)

```

Comment:
As we're now DI-ing this dbcontext, we can just define the audit method straight on here :)